packages/authorization-updater/package.json

```diff
@@ -14,6 +14,8 @@
     "format:check": "prettier --check src",
     "format:write": "prettier --write src",
     "start": "node --watch --no-warnings --loader ts-node/esm -r 'dotenv-flow/config' ./src/index.ts",
+    "generate-model": "mkdir -p ./src/model/generated && pnpm openapi-zod-client './open-api/authorization-management.spec.yaml' -o './src/model/generated/api.ts'",
```

Comment:
I'm agree with yout it could be track as script's command improvements.

Reply:
my concern is that from now on this will be taken as "the way" and then we will need to refactor a lot of code.
If the effort for this change is minor, I would suggest to di it inside this PR, of course just for the code related to this service, not for all the others.

If I'm not mistaken, this is the first service that is calling another API, the right setup now will be repaid on the future uses

Reply:
It's correct is the first service that require API invocation, in this case for our usages the folder `/clients` could be a good destination for output. 

**My2Cents**
However the command `openapi-zod-client` above all generates a zod object with API definitions and then provides a `createApiClient` function to eventually build the client, so contrary of command name 😅 this command mainly generate a set of API's models and potentially creates a client. This definition give the m

anyway as you suggest I proceed with renaming destination folder and command name.

Reply:
@galales  this job is heavy and have a lots of changes around packages, I have addressed this issue in a separated [PR](https://github.com/pagopa/interop-be-monorepo/pull/274) branched from this.
if it's ok for you I suggest to go away with this PR and handle this problems in the other.